tests/sentry/tasks/test_unmerge.py

```diff
@@ -273,6 +276,10 @@ def create_message_event(template, parameters):
             (u'sentry:release', u'version', 17),
         ])
 
+        assert features.query(source) == [
+            (source.id, {'message:message:character-shingles': 1.0}),
+        ]
```

Comment:
Eh, think it's fine. It'll also just be a quick change whenever extraction is changed.